assets/js/frontend/single-product.js

```diff
@@ -228,10 +261,26 @@ jQuery( function( $ ) {
 			};
 
 			// Initializes and opens PhotoSwipe.
-			var gallery = new PhotoSwipe( pswpElement, PhotoSwipeUI_Default, items.items, options );
+			var gallery = new PhotoSwipe( pswpElement, PhotoSwipeUI_Default, items, options );
 			gallery.init();
-		}
+		};
+
+		this.init();
+	};
+
+	/**
+	 * Function to call wc_product_gallery on jquery selector.
+	 */
+	$.fn.wc_product_gallery = function( args ) {
+		new Product_Gallery( this, args );
+		return this;
 	};
 
```

Comment:
Third parties can use this to initialize gallery functionality on any jQuery element.